packages/react/src/components/LinkWithIcon/LinkWithIcon.js

```diff
@@ -43,6 +48,11 @@ LinkWithIcon.propTypes = {
    */
   href: PropTypes.string,
 
+  /**
+   * Display icon to left of link.
+   */
+  iconOnLeft: PropTypes.bool,
```

Comment:
```suggestion
  iconPlacement: PropTypes.oneOf(['left', 'right']),
```

suggestion(non-blocking): Carbon has had to make some breaking changes when we found a need for the 3rd option for where `bool` props are defined.

Reply:
Good call, thanks @asudoh!